contracts/cw-proposal-single/src/tests.rs

```diff
@@ -3847,82 +3858,6 @@ fn test_migrate_from_compatible() {
     assert_eq!(config, new_config);
 }
 
-#[test]
-fn test_migrate_from_beta() {
```

Comment:
yeah, I don't think there's a good way to manipulate state with multi-test. Let's keep the mock test for now and add an integration test later

Reply:
made an issue: https://github.com/DA0-DA0/dao-contracts/issues/456